test/scripts/openshift-ci/setup-e2e-tests.sh

```
@@ -15,17 +15,22 @@
 set -o errexit
 set -o nounset
 set -o pipefail
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-PROJECT_ROOT="$(cd "$SCRIPT_DIR/../../.." && pwd)"
-
 source "$SCRIPT_DIR/common.sh"
+PROJECT_ROOT="$(find_project_root "$SCRIPT_DIR")"
+
+readonly MARKERS="${1:-raw}"
+readonly PARALLELISM="${2:-1}"
```

Shellcheck (0.11.0)

[warning] PARALLELISM appears unused. Verify use (or export if used externally).

(SC2034)